### PR TITLE
Docs: true/false with boolean values

### LINKED ISSUE #78925: Docs: various files - replace boolean yes/no with true/false
### Summary

Based on the [steering committee vote to use true/false for booleans,](https://github.com/ansible-community/community-topics/discussions/120) we are going through the guides to adapt to this change.

This issue requests these changes to the files listed in a followon comment.

Changes are: change `yes` to `true`  and `no` to `false`

must be lowercase. Please open one PR to handle these changes. It should impact 6 files. NOTE - ansibot does not like PRs over 50 files.

The following grep was used to create the list.
`grep -R '\: \(yes\|no\)$' --exclude-dir=locales`

NOTE: there are multiple issues open to change these booleans so please limit your changes to the list below so we do not have clashing PRs.

### Issue Type

Documentation Report

### Component Name

docs/docsite/rst/dev_guide/developing_modules_general_windows.rstt

### Ansible Version

```console
$ ansible --version
2.14
```

### Configuration

```console
# if using a version older than ansible-core 2.12 you should omit the '-t all'
$ ansible-config dump --only-changed -t all
none
```

### OS / Environment

none

### Additional Information

none

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by prayatharth**
##### SUMMARY

Fixes #78925 

##### **ISSUE TYPE**

- Docs Pull Request

### CODE REVIEW

File: docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst [hunk 1]
```diff
@@ -110,7 +110,7 @@ We will add the following code to the file.
   - name: Test for new_option, create new user WITHOUT the attribute
     postgresql_user:
       name: test_user
-      add_attribute: no
+      add_attribute: false
```
Comment (Akasurde) [inline docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst]: `add_attribute` is not a valid parameter for `postgresql_user` (atleast I was not able to find it in the docs). Can you please remove it/verify if it exists?

File: docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst [hunk 2]
```diff
@@ -110,7 +110,7 @@ We will add the following code to the file.
   - name: Test for new_option, create new user WITHOUT the attribute
     postgresql_user:
```
Comment (Akasurde) [inline docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst]:
```suggestion
    community.postgresql.postgresql_user:
```

File: docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst (hunk 1 above)
Comment (Akasurde) [reply docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst]: cc @Andersson007

File: docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst [hunk 3]
```diff
@@ -131,7 +131,7 @@ We will add the following code to the file.
   - name: Test for new_option, create new user WITH the attribute
     postgresql_user:
```
Comment (Akasurde) [inline docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst]:
```suggestion
    community.postgresql.postgresql_user:
```

File: docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst [hunk 4]
```diff
@@ -131,7 +131,7 @@ We will add the following code to the file.
...
       name: test_user
-      add_attribute: yes
+      add_attribute: true
```
Comment (Akasurde) [inline docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst]: Same here.

File: docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst (hunk 3 above)
Comment (Akasurde) [reply docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst]: Can you please change all occurrences by adding FQCN `community.postgresql.` to postgresql modules?

File: docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst [hunk 5]
```diff
@@ -153,16 +153,16 @@ Then we :ref:`run the tests<collection_run_integration_tests>` with ``postgresql
 
 In reality, we would alternate the tasks above with the same tasks run with the ``check_mode: yes`` option to be sure our option works as expected in check-mode as well. See :ref:`Recommendations on coverage<collection_integration_recommendations>` for details.
 
-If we expect a task to fail, we use the ``ignore_errors: yes`` option and check that the task actually failed and returned the message we expect:
+If we expect a task to fail, we use the ``ignore_errors: true`` option and check that the task actually failed and returned the message we expect:
 
 .. code-block:: yaml
 
   - name: Test for fail_when_true option
     postgresql_user:
```
Comment (Akasurde) [inline docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst]:
```suggestion
    community.postgresql.postgresql_user:
```

File: docs/docsite/rst/tips_tricks/sample_setup.rst [hunk 6]
```diff
@@ -191,7 +191,7 @@ Ansible loads any file called ``main.yml`` in a role sub-directory. This sample
       service:
         name: ntpd
         state: started
-        enabled: yes
+        enabled: true
```
Comment (Akasurde) [inline docs/docsite/rst/tips_tricks/sample_setup.rst:194]: Since you are here, can you please add FQCN to all the modules' examples? I am OK doing this in the same PR or a separate PR.

**Review by Akasurde — CHANGES_REQUESTED** [review]

File: docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst (hunk 1 above)
Comment (Andersson007) [reply docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst]: i also don't see it, should be removed then, @Akasurde thanks

Comment (prayatharth) [reply docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst]: @Akasurde  So you want me to remove  `add_attribute` here for the `postgresql_user`  right?

File: docs/docsite/rst/tips_tricks/sample_setup.rst (hunk 6 above)
Comment (prayatharth) [reply docs/docsite/rst/tips_tricks/sample_setup.rst:194]: Hey! @Akasurde since I am new to all this I am not very used to technical terms here but I am eager to learn .

Basically you want me to replace `postgresql_user` with `community.postgresql.postgresql_user` right ?

File: docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst (hunk 1 above)
Comment (samccann) [reply docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst]: @Akasurde @Andersson007 - is it as simple as removing that one line, or does that impact the rest of the example? We can make the suggested edits to remove in this PR, but please review the full example and let us know if more needs to change (besides FQCN).

File: docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst (hunk 3 above)
Comment (samccann) [reply docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst:132]: @Akasurde I'd like to open a separate issue for this. Partly because this is a hacktoberfest issue so I'd prefer to keep the scope limited to boolean fixes as much as possible. And partly because if the problem is here, it might be in a few other of the community example files so I want to scan them all and then open a new issue to track fixing them all.

**Comment by prayatharth** [comment]
Hi @samccann thank you for letting me get the experience of opensource .

Do you think is there anything that I can do to what contributors pointed out ?

**Comment by samccann** [comment]
@prayatharth if you are up for doing the FQCN change @Akasurde talked about above, that would be great! Then we just need to wait on the answer to my other question on if we can just remove the two lines he mentioned about `add_attribute: true`.

File: docs/docsite/rst/community/collection_contributors/collection_integration_about.rst [hunk 7]
```diff
@@ -139,7 +139,7 @@ Example:
     abstract_module:
         ...
     register: result
-    ignore_errors: yes
+    ignore_errors: true
```
Comment (samccann) [inline docs/docsite/rst/community/collection_contributors/collection_integration_about.rst]:
```suggestion
```

File: docs/docsite/rst/community/collection_contributors/collection_integration_add.rst [hunk 8]
```diff
@@ -213,7 +213,7 @@ That is enough for our very basic example.
 .. code-block:: yaml
 
   - name: Test postgresql_info module
-    become: yes
+    become: true
     become_user: postgres
     postgresql_info:
```
Comment (samccann) [inline docs/docsite/rst/community/collection_contributors/collection_integration_add.rst]: Can you make the same changes to this file as well? to change `postgresql_info` to `community.postgresql.postgresql_user`?

Basically, we have to prepend all occurrences of `postgresql_` with `community.postgresql.postresql_` on all these files.  The shorter name is an old convention that is outdated.

File: docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst [hunk 9]
```diff
@@ -108,9 +108,9 @@ We will add the following code to the file.
   # https://github.com/ansible-collections/community.postgresql/issues/NUM
   # We should also run the same tasks with check_mode: yes. We omit it here for simplicity.
   - name: Test for new_option, create new user WITHOUT the attribute
-    postgresql_user:
+    community.postgresql.postgresql_user:
       name: test_user
-      add_attribute: no
+      add_attribute: false
```
Comment (samccann) [inline docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst]:
```suggestion
```

File: docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst [hunk 10]
```diff
@@ -129,9 +129,9 @@ We will add the following code to the file.
         - result.query_result.rowcount == 1
 
   - name: Test for new_option, create new user WITH the attribute
-    postgresql_user:
+    community.postgresql.postgresql_user:
       name: test_user
-      add_attribute: yes
+      add_attribute: true
```
Comment (samccann) [inline docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst]:
```suggestion
```

**Review by samccann — COMMENTED** [review]
a few other comments. This is looking good!

File: docs/docsite/rst/community/collection_contributors/collection_integration_add.rst (hunk 8 above)
Comment (prayatharth) [reply docs/docsite/rst/community/collection_contributors/collection_integration_add.rst]: @samccann  just making sure you want me to change `postgresql_info`  to `community.postregsql.postgresql_user` not 
`community.postregsql.postgresql_info` right ?

Comment (Akasurde) [reply docs/docsite/rst/community/collection_contributors/collection_integration_add.rst]: `community.postgresql.postgresql_info`

File: docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst (hunk 1 above)
Comment (Akasurde) [reply docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst]: Let us tackle this in separate PR since the whole example is talking about creating a user without attributes.

File: docs/docsite/rst/tips_tricks/sample_setup.rst (hunk 6 above)
Comment (Akasurde) [reply docs/docsite/rst/tips_tricks/sample_setup.rst:194]: Yes, you are right. Ansible now uses Fully Qualified Collection Names (FQCN) to refer to the modules. So `service` will be referred to as `ansible.builtin.service`. You can check this by using the `ansible-doc` command - 

```
# ansible-doc service
...
> ANSIBLE.BUILTIN.SERVICE    (/src/ansible/lib/ansible/modules/service.py)
...
```

File: docs/docsite/rst/community/collection_contributors/collection_integration_add.rst [hunk 11]
```diff
@@ -215,7 +215,7 @@ That is enough for our very basic example.
   - name: Test postgresql_info module
     become: true
     become_user: postgres
-    postgresql_info:
+    community.postregsql.postgresql_info:
```
Comment (Akasurde) [inline docs/docsite/rst/community/collection_contributors/collection_integration_add.rst]:
```suggestion
    community.postgresql.postgresql_info:
```

File: docs/docsite/rst/community/collection_contributors/collection_integration_about.rst [hunk 12]
```diff
@@ -139,7 +139,7 @@ Example:
     abstract_module:
         ...
     register: result
-    ignore_errors: true
+
```
Comment (Akasurde) [inline docs/docsite/rst/community/collection_contributors/collection_integration_about.rst]:
```suggestion
```

File: docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst [hunk 13]
```diff
@@ -110,7 +110,7 @@ We will add the following code to the file.
   - name: Test for new_option, create new user WITHOUT the attribute
     community.postgresql.postgresql_user:
       name: test_user
-      add_attribute: false
+
```
Comment (Akasurde) [inline docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst]: I would say do these changes in separate PR.

File: docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst [hunk 14]
```diff
@@ -131,7 +131,7 @@ We will add the following code to the file.
   - name: Test for new_option, create new user WITH the attribute
     community.postgresql.postgresql_user:
       name: test_user
-      add_attribute: true
```
Comment (Akasurde) [inline docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst]: Do these changes in a separate PR.

**Review by Akasurde — CHANGES_REQUESTED** [review]

File: docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst (hunk 13 above)
Comment (prayatharth) [reply docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst]: Okay

File: docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst (hunk 14 above)
Comment (prayatharth) [reply docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst]: @Akasurde then there needs to be a separate issue to be raised for this in order for me create a PR ?

Comment (prayatharth) [reply docs/docsite/rst/community/collection_contributors/collection_integration_updating.rst]: @Akasurde  done in separate PR #79019

**Comment by prayatharth** [comment]
@samccann  if everything looks okay so can we close this issue and I have created a separate PR #79019  as suggested for `add_attribute`

File: docs/docsite/rst/community/collection_contributors/collection_integration_about.rst [hunk 15]
```diff
@@ -27,7 +27,7 @@ Let's say we want to test the ``postgresql_user`` module invoked with the ``name
 .. code-block:: yaml
 
   - name: Create PostgreSQL user and store module's output to the result variable
-    postgresql_user:
+    communnity.postgresql.postgresql_user:
```
Comment (samccann) [inline docs/docsite/rst/community/collection_contributors/collection_integration_about.rst]:
```suggestion
    community.postgresql.postgresql_user:
```

File: docs/docsite/rst/community/collection_contributors/collection_integration_about.rst [hunk 16]
```diff
@@ -37,7 +37,7 @@ Let's say we want to test the ``postgresql_user`` module invoked with the ``name
         - result is changed
 
   - name: Check actual system state with another module, in other words, that the user exists
-    postgresql_query:
+    communnity.postgresql.postgresql_query:
```
Comment (samccann) [inline docs/docsite/rst/community/collection_contributors/collection_integration_about.rst]:
```suggestion
    community.postgresql.postgresql_query:
```

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @prayatharth for the Ansible docs PR and welcome to Ansible!

**Comment by prayatharth** [comment]
Thank You @samccann   for the opportunity and for being so supportive throughout the way.
